manifests/apache/vhost.pp

```diff
@@ -24,6 +24,7 @@
 class puppetboard::apache::vhost (
   String[1] $vhost_name,
   Stdlib::Unixpath $wsgi_alias                 = '/',
+  Stdlib::IP::Address $ip                      = '127.0.0.1',
```

Comment:
This breaks backward compatibility.  Maybe make this optional?
```suggestion
  Optional[Stdlib::IP::Address] $ip            = undef,
```

Reply:
replaced as requested with undef, added parameter description, and renamed commit

Reply:
Thanks!